src/features/projectsV2/ProjectDetails/components/OtherInterventionInfo.tsx

```diff
@@ -0,0 +1,216 @@
+                console.error('Error parsing JSON:', error);
+              }
+            } else {
+              // If not a JSON string, add the `label` and `value` directly
+              cleanedData.push({
+                key: value.label,
+                value: value.value,
+              });
+            }
+          }
+        }
+      }
+    );
+  }
+
+  // Return the array of cleaned key-value pairs
+  return cleanedData;
+};
+
+interface Props {
+  plantLocationInfo: OtherInterventions | null;
+  isMobile: boolean;
+  setSelectedSamplePlantLocation: SetState<SamplePlantLocation | null>;
+}
+
+const OtherInterventionInfo = ({
+  plantLocationInfo,
+  isMobile,
+  setSelectedSamplePlantLocation,
+}: Props) => {
+  const tProjectDetails = useTranslations('ProjectDetails');
+
+  const { totalTreesCount } = useMemo(() => {
+    const totalTreesCount =
+      plantLocationInfo &&
+      plantLocationInfo.plantedSpecies &&
+      plantLocationInfo.plantedSpecies.length > 0
+        ? plantLocationInfo.plantedSpecies.reduce(
+            (sum, species: { treeCount: number }) => sum + species.treeCount,
+            0
+          )
+        : 0;
+    return { totalTreesCount };
+  }, [plantLocationInfo, plantLocationInfo?.type]);
+
+  const sampleInterventionSpeciesImages = useMemo(() => {
+    if (plantLocationInfo && plantLocationInfo.sampleInterventions.length > 0) {
+      const result =
+        plantLocationInfo.sampleInterventions &&
+        plantLocationInfo.sampleInterventions.map((item) => {
+          return {
+            id: item.coordinates[0].id,
+            image: item.coordinates[0].image ?? '',
+            description: tProjectDetails('sampleTreeTag', { tag: item.tag }),
+          };
+        });
+      return result;
+    }
+  }, [plantLocationInfo]);
+
+  const shouldDisplayImageCarousel =
+    sampleInterventionSpeciesImages !== undefined &&
+    sampleInterventionSpeciesImages?.length > 0;
+
+  const cleanedPublicMetadata = createCardData(plantLocationInfo);
+
+  const content = [
+    <>
+      <InterventionHeader
+        plHid={plantLocationInfo?.hid}
+        interventionType={plantLocationInfo?.type}
+        key="interventionHeader"
+      />
+      {shouldDisplayImageCarousel && (
+        <ImageSlider
+          key="imageSlider"
+          images={sampleInterventionSpeciesImages}
+          type="coordinate"
+          isMobile={isMobile}
+          imageSize="large"
+          allowFullView={!isMobile}
+        />
+      )}
+    </>,
+    cleanedPublicMetadata.length > 0 && (
+      <>
+        <OtherInterventionInfoHeader
+          key="interventionHeader"
+          plantDate={plantLocationInfo?.interventionStartDate}
+        />
+        <OtherInterventionMetadata
+          key="plantingDetails"
+          metadata={cleanedPublicMetadata}
+          plantDate={plantLocationInfo?.interventionStartDate}
+          type={plantLocationInfo?.type}
+        />
+      </>
+    ),
```

Comment:
This logic will hide the intervention date if cleanedMetadata is empty, perhaps this should be tweaked.

Reply:
This is fixed, but the mobile UI needs to be adjusted a bit.

<img width="1440" alt="Screenshot 2025-01-15 at 2 05 08 PM" src="https://github.com/user-attachments/assets/a52a6dcd-df01-4ee6-ac5c-f1563ae8644a" />